ports/nrf/drivers/bluetooth/ble_drv.c

```diff
@@ -62,6 +63,10 @@
 #define BLE_SLAVE_LATENCY            0
 #define BLE_CONN_SUP_TIMEOUT         MSEC_TO_UNITS(4000, UNIT_10_MS)
 
+#if (BLUETOOTH_SD == 132 && BLE_API_VERSION == 4)
+#define GATT_MTU_SIZE_DEFAULT BLE_GATT_ATT_MTU_DEFAULT
```

Comment:
you might want to use something like:

```c
#if !defined(GATT_MTU_SIZE_DEFAULT) && defined(BLE_GATT_ATT_MTU_DEFAULT)
#define GATT_MTU_SIZE_DEFAULT BLE_GATT_ATT_MTU_DEFAULT
#endif

Or maybe avoid redefining this, I think `GATT_MTU_SIZE_DEFAULT` is provided by the Nordic header files?

Reply:
that is the catch here. The new define has new name, in v5 this is called BLE_GATT_ATT_MTU_DEFAULT and not legacy GATT_MTU_SIZE_DEFAULT. So this basically is a redefinition of this value to unify it.

Reply:
Ah, that's the issue. Still, we can simply `#define` one to the other if the one isn't provided, like my code above does

Reply:
Agree. Your proposal is more future proof. There will be a similar case adding support for nrf52840 SD140v6, which does not provide BLE_GAP_ADV_MAX_SIZE. I'll adopt your solution there as well.